### DISCUSSION TITLE
AHAHAHAHHAHAHAHAHHA

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#414] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I HAVE TWO ANSWERS NOW AHAHAHA IM 4TH IN THE ANSWER ELADERBOARD GET NEED EVERYONE BELOW ME AHAHAHAHAHAHHAHAHAHAHAHHA IM STILL BEING SPIED ON WITH DYKNOW THOUGH AHAHAHAHAHSskjdgf wdswj dksjdgvj c390*()WUF [fe\]":<
V:
'.,/. `

oh yeah also update the desc of the ext remover "Bookmarklet exploit that can force-disable extensions installed on Chrome. Also has a very fancy GUI to manage all extensions!" is kinda outdated considering this has went from not getting spied on by games to breaking the law and unenrolling your chromebook...

**Comment by Aka-but-nice** [comment]
lmao

**Comment by Alex2Mayhem** [comment]
Are you okay? Do you need mental help?
